### PR TITLE
Tt 990 new registration journey individual

### PR DESCRIPTION
**Pull request by gauravdott**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if changing the UI) Add a printscreen to the PR

Enrol-1

Enrol-2

Enrol-3

Enrol-4

Enrol-5

Enrol-6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ## Pre-release
 
 ### Implemented enhancements
+=======
```
Comment (richtier) [inline CHANGELOG.md]: for consistency please remove ====== :)

Comment (gauravdott) [reply CHANGELOG.md]: Ah yes, the residual merge conflict. Done!

File: enrolment/templates/enrolment/success-individual.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{% extends 'enrolment/base.html' %}
+
+
+{% load static from staticfiles %}
+{% load message_box_with_icon breadcrumbs from directory_components %}
+
+{% block content %}
+    <section class="container">
+      {% breadcrumbs 'Create a great.gov.uk account' %}
+          <a href="{{ services_urls.exred }}">Home</a>
```
Comment (richtier) [inline enrolment/templates/enrolment/success-individual.html]: please use `services_urls.great_domestic` (exred is the backwards compatible name, but the service has not been called exred ins  while :) )

https://github.com/uktrade/directory-components/blob/master/directory_components/context_processors.py#L94

File: enrolment/templates/enrolment/widgets/radioselect_helpertext.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="multiple-choice" {% if widget.attrs.helperTexts %}
```
Comment (richtier) [inline enrolment/templates/enrolment/widgets/radioselect_helpertext.html]: this file is not needed :)

File: enrolment/tests/test_views.py [hunk 4]
```diff
@@ -44,6 +45,18 @@ def submit_sole_trader_step(client):
     )
 
 
+@pytest.fixture
+def submit_individual_step(client):
+
+    # The WizardView converts view class name from camelcase to
+    # snake case to get the view name
```
Comment (richtier) [inline enrolment/tests/test_views.py]: comment not very helpful :)

Comment (gauravdott) [reply enrolment/tests/test_views.py]: Yes, it's not needed anymore. It was added when we were converting the camelcase manually to snakecase. Forgot to remove it. Removed now!

File: makefile [hunk 5]
```diff
@@ -69,6 +73,8 @@ DEBUG_SET_ENV_VARS := \
 	export DIRECTORY_CH_SEARCH_CLIENT_BASE_URL=http://127.0.0.1:8012; \
 	export DIRECTORY_FORMS_API_BASE_URL=http://forms.trade.great:8011; \
 	export DIRECTORY_API_CLIENT_BASE_URL=http://api.trade.great:8000; \
+	export DIRECTORY_FORMS_API_API_KEY=n2DxBRFfcFS8k1EYyAXYBK4eWhApkxEuLIHwjhxJprOFz57By1AhcRWGgs6YmU8g; \
+	export DIRECTORY_FORMS_API_SENDER_ID=1d99e352-3f2c-4c52-a843-55cda003a32a; \
```
Comment (richtier) [inline makefile]: please remove and cycle the keys if this is secrets from dev/stage/prod

File: enrolment/templates/enrolment/widgets/radioselect_helpertext.html (hunk 3 above)
Comment (gauravdott) [reply enrolment/templates/enrolment/widgets/radioselect_helpertext.html]: :) Ah yes! Done!

File: makefile (hunk 5 above)
Comment (gauravdott) [reply makefile]: Done!

File: enrolment/templates/enrolment/success-individual.html (hunk 2 above)
Comment (gauravdott) [reply enrolment/templates/enrolment/success-individual.html]: Done!

**Pull request merged**